features/steps/partcad-cli/commands/status.py

```
@@ -0,0 +1,17 @@
+import os
+import yaml
+from behave import given, when, then
```

Ruff (0.8.0)

`behave.given` imported but unused

Remove unused import

(F401)